src/schema/v2/me/bidder_position.ts

```diff
@@ -41,6 +41,11 @@ export const BidderPosition: GraphQLFieldConfig<void, ResolverContext> = {
             }
           } else if (position.processed_at && !anyReserveMet(position)) {
             status = "RESERVE_NOT_MET"
+          } else if (
```

Comment:
I might move this up in this waterfall of conditionals (maybe make it the first one)? 

I'd worry that one of these other conditionals ahead of it  _may_ match. With this error state we know right away that the `status` should be `SALE_CLOSED`.

Reply:
that's a good point, hadn't considered that, will do!